app/code/Magento/Config/App/Config/Source/RuntimeConfigSource.php

```diff
@@ -54,13 +54,13 @@ public function __construct(
      * Get initial data.
      *
      * @param string $path Format is scope type and scope code separated by slash: e.g. "type/code"
-     * @return array
+     * @return mixed
      * @since 100.1.2
      */
     public function get($path = '')
     {
         $data = new DataObject($this->loadConfig());
-        return $data->getData($path) ?: [];
+        return !is_null($data->getData($path)) ? $data->getData($path): [];
```

Comment:
Hi @shikhamis11!
Thank you for your contribution!
Could we add these cases to the unit tests `\Magento\Config\Test\Unit\App\Config\Source\RuntimeConfigSourceTest` ?

Reply:
sure let me check